src/analysis/special_functions/log.lean

```diff
@@ -195,6 +195,11 @@ begin
     ...           = log x / x * y - log x : by ring,
 end
 
+/-- The real logarithm in a given base. -/
+noncomputable def log_base (b x : ℝ) : ℝ := log x / log b
+
+@[simp] lemma log_div_log {b x : ℝ} : log x / log b = log_base b x := rfl
```

Comment:
If you add this lemma, then you should add lots of API lemmas about `log_base` (continuity in both arguments, differentiability, basic lemmas like `log_base b (x * y)`).

Reply:
Ok, I've taken the approach of copying `log.lean` into `logb.lean` and changing all the `log`s to `logb`s and the `exp`s to `rpow`s. This covers the basic lemmas you mention. An API for continuity and differentiability might be harder, both because some of the lemmas for `exp` used in `log.lean` don't seem to exist for `rpow`, and because I'm less familiar in general with how mathlib formalizes these things. 

For these reasons I'm inclined to say these things are best left for another PR. If you disagree, feel free to say so and unresolve this, and I'll try to start learning about filters.